fbchat/_client.py

```diff
@@ -976,6 +976,42 @@ def getUserActiveStatus(self, user_id):
         """
         return self._buddylist.get(str(user_id))
 
+    def _fetchImages(self, thread_id):
+        data = {"id": thread_id, "first": 12}
+        j = self.graphql_request(_graphql.from_query_id("515216185516880", data))
+        to_continue = True
+        while to_continue:
+            page_info = j[thread_id]["message_shared_media"]["page_info"]
+            end_cursor = page_info.get("end_cursor")
+            try:
+                yield j[thread_id]["message_shared_media"]["edges"][0]
+                del j[thread_id]["message_shared_media"]["edges"][0]
+            except IndexError:
+                if page_info.get("has_next_page"):
+                    data["after"] = end_cursor
+                    j = self.graphql_request(
+                        _graphql.from_query_id("515216185516880", data)
+                    )
+                else:
+                    to_continue = False
```

Comment:
Use `break` or `return` instead of keeping track of a variable like this!

Reply:
Yup, changed it when i merged both function together.